### PR TITLE
CP-367 Do not minify:js in dist by default

### PR DESCRIPTION
**Pull request by maxwellpeterson-wf**
## Description

This change is to align with CP team's default library workflow.
We will distribute unminified library files in `dist/`, and a minified globalBundle with sourcemaps in `lib/` for Dart consumption. That minification is done by jspm itself during the bundling process.

## Testing

install that branch of wGulp in some project and run `gulp dist` and make sure it doesn't run the minify:js task

@trentgrover-wf 
@evanweible-wf

### CODE REVIEW

**Comment by rmconsole-wf** [comment]
When this pull is merged I will not be able to function
		properly because no ticket is included in the title.

***
*Last updated on Tuesday, January 20 04:04 PM CST*

**Comment by evanweible-wf** [comment]
+1

File: src/gulpconfig.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
         "build": ["clean", "lint", "tsd", "jsx", "tsc", "copy:html", "copy:js", "coffee", "livescript", "sass"],
         "bundle": ["clean:dist", "build"],
         "default": ["clean", "build", "test", "analyze", "jsdoc", "dist"],
-        "dist": ["clean", "build", "bundle", "libraryDist", "minify"],
+        "dist": ["clean", "build", "bundle", "libraryDist", "minify:css"],
```
Comment (trentgrover-wf) [inline src/gulpconfig.json]: why minify:css if we're killing minify of js?

Comment (maxwellpeterson-wf) [inline src/gulpconfig.json]: Figured for the most part apps (not libraries) would have actual css files and they would likely want minified versions of them by default.

I'm fine with removing that also, though.

**Comment by trentgrover-wf** [comment]
+1
@jayudey-wf ready for merge

**Comment by jayudey-wf** [comment]
QA Resource Approval: +10

- [ ] Testing instruction
- [ ] Dev +1's
- [ ] Dev/QA +10 with detail of what was tested
 - verified there were no minified js 
- [ ] Unit test created/updated
- [ ] All unit tests pass

Merging into master.

**Pull request merged**